help/destinations/catalog/email-marketing/salesforce-pardot.md

```diff
@@ -0,0 +1,203 @@
+
+## Prerequisites {#prerequisites}
+
+Refer to the sections below for any prerequisites that have to be setup / information that is to be gathered before working with the [!DNL Salesforce Pardot] destination.
+
+### Prerequisites in Experience Platform {#prerequisites-in-experience-platform}
+
+Before activating data to the [!DNL Salesforce Pardot] destination, you must have a [schema](/help/xdm/schema/composition.md), a [dataset](https://experienceleague.adobe.com/docs/platform-learn/tutorials/data-ingestion/create-datasets-and-ingest-data.html?lang=en), and [segments](https://experienceleague.adobe.com/docs/platform-learn/tutorials/segments/create-segments.html?lang=en) created in [!DNL Experience Platform].
+
+>[!TIP]
+>
+>The [!DNL Salesforce Pardot] destination uses the selected segment names, to create custom prospect fields in [!DNL Salesforce Pardot], and update them with the Experience Platform segment status. The destination automatically replaces whitespace or special characters with the underscore `(_)` character as a separator to comply with [!DNL Salesforce Pardot] requirements.
+>However, as a recommendation, when creating a segment, do not include whitespace or special characters in the segment name. Instead, use the underscore `(_)` character as a separator. For example instead of naming your segment as `Salesforce Pardot Segment 1` use `Salesforce_Pardot_Segment_1`.
+
+### Prerequisites in [!DNL Salesforce Pardot] {#prerequisites-destination}
+
+Note the following prerequisites in order to export data from Platform to your [!DNL Salesforce Pardot] account:
+
+#### You need to have a [!DNL Salesforce Pardot] account {#prerequisites-account}
+
+A [!DNL Salesforce Pardot] account with a subscription to the [Marketing Cloud Account Engagement](https://www.salesforce.com/products/marketing-cloud/marketing-automation/) product is mandatory to proceed.
+
+>[!IMPORTANT]
+>
+>Your account should also have the [Account Engagement Administrator role](https://help.salesforce.com/s/articleView?id=sf.pardot_fields_create_custom_field.htm&type=5) and the [!DNL Salesforce] admin must give permissions to the [!DNL Account Engagement admin] user that allow access to [!DNL Account Engagement Lightning App].
+
+Reach out to [[!DNL Salesforce] Support](https://www.salesforce.com/company/contact-us/?d=cta-glob-footer-10) or your [!DNL Salesforce] account administrator if you do not have an account, or the account is missing the [!DNL Marketing Cloud Account Engagement] subscription or the [!DNL Account Engagement Administrator role].
+
+#### Configure a connected app within [!DNL Salesforce] {#prerequisites-connected-app}
+
+You can follow the [!DNL Salesforce] documentation [here](https://help.salesforce.com/s/articleView?id=sf.pardot_sf_connector_setup_v2_parent.htm&type=5) or alternatively check the steps below.
```

Comment:
I think an intro would be good here, like: 

To connect Experience Platform to Salesforce, you need to configure a connected app within [!DNL Salesforce]. (does this make sense?)